ontoconv/pipelines.py

```diff
@@ -7,10 +7,30 @@
 from otelib import OTEClient
 from tripper import OTEIO, RDF, Triplestore
 from tripper.convert import load_container, save_container
+from tripper.convert.convert import BASIC_RECOGNISED_KEYS
+
+from ontoconv.attrdict import AttrDict
 
 # Get rid of FutureWarning from csv.py
 warnings.filterwarnings("ignore", category=FutureWarning)
 
+# Extend the recognised keys used by tripper.convert
+RECOGNISED_KEYS = BASIC_RECOGNISED_KEYS.copy()
+RECOGNISED_KEYS.update(
+    {
+        "aiida_plugin": "http://open-model.eu/ontologies/oip#AiidaPlugin",
+        "command": "http://open-model.eu/ontologies/oip#Command",
+        "install_command": (
+            "http://open-model.eu/ontologies/oip#InstallCommand"
+        ),
```

Comment:
In which ontology are these concepts defined?

Reply:
No ontology has been created yet. Added an issue: https://github.com/H2020-OpenModel/OntoConv/issues/9